packages/integration-test/src/stability.spec.ts

```diff
@@ -409,14 +480,16 @@ describe("stability check, assess a solution, reassess the solution, check all s
     const results = await runThroughSolution(
       solutionPath,
       "inplace",
-      "netcoreapp3.1"
+      "netcoreapp3.1",
+      false,
+      false
     );
     await validateHighLevelResults(results, [
-      "0 of 1",
-      "2 of 6",
-      "50 of 81",
-      "0",
-      "(21)",
+      "1 of 1",
```

Comment:
one thing to watch out for here is that, the pipeline will use unported solutions, so if you're running the integ tests multiple times, it might be running against a ported solution so double check this "1 of 1" as well as other values just in case

Reply:
Thanks for finding this. Yes, you're right. The number hasn't changed. Back to original code.